tests/binaries/src/hello_world.c

```diff
@@ -0,0 +1,6 @@
+#include <stdio.h>
+
```

Comment:
I assume this file was compiled with `gcc -o hello_world hello_world.c` but let's comment anyway to be consistent